Sources/FluentKit/Properties/Timestamp.swift

```diff
@@ -130,3 +171,31 @@ extension Schema {
         query.filters.append(.group([isNull, isFuture], .or))
     }
 }
+
+
+public protocol TimestampFormatter {
+    func string(from date: Date) -> String
```

Comment:
Would it make sense to return `Encodable` here? For example `TimestampFormat.unix` would want to return an `Int`.